app/api/app_last_active.ts

```diff
@@ -0,0 +1,50 @@
+import { EnvironmentName, getEnvironment } from '@environment'
```

Comment:
do we still need this file?

Reply:
Nope, will delete all the `persistent layer` for `last active` after some manual test across platforms are done. Currently the usage are commented out.